Common/Product/SharedProject/CommonProjectNode.cs

```diff
@@ -636,11 +636,11 @@ private void UpdateShowAllFiles(HierarchyNode node, bool enabled)
         {
             bool? showAllFiles = null;
             var showAllFilesSetting = showAllFilesValue.Trim();
-            if (String.Equals(showAllFilesSetting, CommonConstants.ProjectFiles))
+            if (StringComparer.OrdinalIgnoreCase.Equals(showAllFilesSetting, CommonConstants.ProjectFiles))
```

Comment:
Equals by default is:
ordinal (case-sensitive and culture-insensitive) comparison.
You're now ignoring case here and below - is this deliberate?